app/Dashboard.html

```diff
@@ -53,9 +144,39 @@
                 "token":"${token}"
             }
           `
-        });
-        return false;
-      }
-    </script>
-  </body>
+    }).then(function (response) {
+        if (response.ok) {
+          toastr.success("Blog Post " + response.statusText);
+          form.reset();
+        } else {
+          toastr.warning("There was a problem submitting the blog post : " + response.statusText);
```

Comment:
Not sure if I know what you mean by "template strings". Would you like to submit a commit with what you have in mind?